step.yml

```diff
@@ -0,0 +1,51 @@
+title: Decrypt file
+summary: This step decrypts a file encrypted with `gpg`, using the default AES128 symmetric cypher.
+description: |
+
+    You can use this step to decrypt a file in your repository or stored on bitrise.io or other provider (with the File Downloader step).
+
+    How to create an encrypted file:
+
+    1. Create a secure passphrase of at least 22 alphanumeric characters: `pwgen -s 22 1`
+    2. Encrypt file: `gpg -c my_secret.txt`
+    3. Upload file (`my_secret.txt.gpg`) to bitrise.io or to any other provider.
+website: https://github.com/bitrise-steplib/bitrise-step-decrypt-file
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-decrypt-file
+support_url: https://github.com/bitrise-steplib/bitrise-step-decrypt-file/issues
+host_os_tags:
+  - osx-10.10
+  - ubuntu-16.04
+type_tags:
+  - utility
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+deps:
+  brew:
+  - name: gpg
+  apt_get:
+  - name: gpg
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/bitrise-step-decrypt-file
+inputs:
+  - decrypt_passphrase: $FILE_DECRYPT_PASSPHRASE
+    opts:
+      title: "Passphrase for file decryption"
+      description: |
+        The passphrase is used as a parameter for gpg.
+      is_sensitive: true
+      is_required: true
+  - encrypted_file_path: $FILE_DECRYPT_ENCRYPTED_FILE_PATH
```

Comment:
Mostly we define default value with envs when any other of our tools define those envs (like BITRISE_PROJECT_PATH is defined by the scanner: https://github.com/bitrise-io/steps-xcode-archive/blob/master/step.yml#L32)